modules/announcements/module.php

```diff
@@ -0,0 +1,172 @@
+<?php
+
+namespace Elementor\Modules\Announcements;
+
+use Elementor\Core\Base\App as BaseApp;
+use Elementor\Modules\Announcements\Classes\Announcement;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Module extends BaseApp {
+
+	/**
+	 * @return bool
+	 */
+	public static function is_active(): bool {
+		return is_admin();
+	}
+
+	/**
+	 * @return string
+	 */
+	public function get_name(): string {
+		return 'announcements';
+	}
+
+	/**
+	 * Render wrapper for the app to load
+	 */
+	private function render_app_wrapper() {
+		?>
+		<div id="e-announcements-root"></div>
+		<?php
+	}
+
+	/**
+	 * Enqueue app scripts
+	 */
+	private function enqueue_scripts() {
+		wp_enqueue_script(
+			'announcements-app',
```

Comment:
```suggestion
			'elementor-announcements-app',
```

Reply:
I'm keeping it because it's part of the webpack